.github/actions/compute-sha256/package.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Since this compiled js code in not human-readable, could you add a comment in the code / PR how we (the reviewers) can verify that the compiled code indeed comes from the .js?
Something as simple as:
```
$ npm install
$ ncc ...
$ diff dist pr_dist
```

This way we add transparency and we can show 3P folks that we do due diligence, the same way we do when adding hashes to SHA256SUM.md for generated release binaries.

(We can also add this in the actions/README.md) in a follow-up PR.

Reply:
I have included README instructions that have included this information.

Reply:
Thanks